src/components/OSCALControlProse.js

```diff
@@ -432,7 +430,7 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
     .map((segment, index) => {
       if (index % 2 === 0) {
         // This is not a parameter placeholder
-        return getTextSegment(segment, index.toString());
+        return getTextSegment(segment);
       }
       if (isEditingStatement) {
         // We're currently editing this statement, so build param input
```

Comment:
I am wondering what the map in this RegExp is trying to accomplish. My interpretation of it is that it renders a textfield component where the parser finds the regular expression (`{{ insert: param, ([0-9a-zA-B-_.]*) }}`) 

Currently there is an issue that I found when the regular expression is embedded within placeholder text. See image below
<img src ="https://user-images.githubusercontent.com/106985986/183512273-7c04b0b5-c6df-4abe-b84d-7d7726c839cf.png">